src/apps/activities/domain/response_values.py

```diff
@@ -37,6 +39,12 @@
 from apps.shared.domain import PublicModel, validate_color, validate_image, validate_uuid
 
 
+class PhrasalTemplateFieldType(str, Enum):
+    SENTENCE = "sentence"
```

Comment:
Should we change this to be `TEXT`, so that it matches its usage as part of `_PhrasalTemplateTextField` ? https://github.com/ChildMindInstitute/mindlogger-backend-refactor/blob/74388887472b347b5059f61fa72c7c5fbdc57464/src/apps/activities/domain/response_values.py#L340-L342

```suggestion
    TEXT = "text"
```

Reply:
Renamed to _PhrasalTemplateSentenceField